apps/meteor/server/features/EmailInbox/EmailInbox_Outgoing.ts

```diff
@@ -216,7 +218,7 @@ callbacks.add(
 				sender: message.u.username,
 				rid: room._id,
 			},
-		);
+		).then((info) => LivechatRooms.updateEmailThreadByRoomId(room._id, info.messageId));
```

Comment:
**nitpick:** if so many chaining of promises is necessary, is it not worthy to make the outer function async?

Reply:
It's two equal .then calls, looks totally fine to me, do you want it changed?

Reply:
At second glance, looks like a fine way to keep the calls in parallel asynchronous.